android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -24,66 +26,48 @@
     private String name;
     private Double latitude;
     private Double longitude;
-    @JsonProperty("location-name")
     private String locationName;
-    @JsonProperty("starts-at")
-    private String startTime;
-    @JsonProperty("ends-at")
-    private String endTime;
+    private String startsAt;
```

Comment:
@arp95 @Shailesh351 This has happened in past that we have ignored migrations and developers were confused as to why their app was crashing on update, if not adding specific migrations, then clear db on upgrade so that at least app does not crash. Sounds good?